### PR TITLE
Backfill share_publicly from shared_publicly

### PR DESCRIPTION
**Pull request by SethPoulsen**
Should not be merged until after #10842 is deployed

### CODE REVIEW

File: apps/prairielearn/src/batched-migrations/20241018171312_questions__share_publicly__backfill.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    await queryAsync(
+      `
+      UPDATE questions AS q
+      SET
+        share_publicly = q.shared_publicly
+      FROM
+        questions AS q
+      WHERE
+        v.id >= $start AND
+        v.id <= $end`,
```
Comment (nwalters512) [inline apps/prairielearn/src/batched-migrations/20241018171312_questions__share_publicly__backfill.ts:24]: I'd probably move this to a separate `.sql` file. I know I've been inconsistent with that in other batched migrations, but I think we should standardize on that like we have for the rest of the codebase (with perhaps the exception of one-line queries like the `SELECT MAX(...)` above).

**Review by nwalters512 — COMMENTED** [review]
#10842 was deployed, so this is no longer blocked.

File: apps/prairielearn/src/batched-migrations/20241018171312_questions__share_publicly__backfill.ts (hunk 1 above)
Comment (SethPoulsen) [reply apps/prairielearn/src/batched-migrations/20241018171312_questions__share_publicly__backfill.ts]: done

**Comment by SethPoulsen** [comment]
I just did an end-to-end test of this locally where I checked out an old commit, loaded data into the database, then checked out this branch and made sure it backfilled correctly and everything. Should be all good to go!

**Pull request merged**